doc/filesize.rst

```diff
@@ -0,0 +1,72 @@
+The Filesize Extension
+======================
+
+The *Filesize* extension provides the ``filesize`` filter.
+
+You need to register this extension before using the ``filesize`` filter::
+
+    $twig->addExtension(new Twig_Extensions_Extension_Filesize());
+
+``filesize``
```

Comment:
E.g. grams, meters, area, time, filesize, ... Am I right? Will be a whole bunch of filters.